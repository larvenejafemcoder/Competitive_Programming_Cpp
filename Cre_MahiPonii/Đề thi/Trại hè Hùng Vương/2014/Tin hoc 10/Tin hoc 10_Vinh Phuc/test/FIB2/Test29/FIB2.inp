98
20 10180
33 261129
29 255383
16 489
12 102
31 227824
26 113457
35 9824617
43 224525461
39 76946879
18 3102
15 681
18 999
37 34347982
11 31
31 509809
8 14
32 1933785
36 1659082
39 101829920
37 8119561
43 101287408
35 13037985
27 126743
18 2051
42 103919788
30 635130
36 16888275
36 19555670
6 12
37 34934166
1 1
44 955979863
45 1793197709
28 153329
19 1517
8 4
17 1996
14 129
32 403985
5 2
12 17
2 2
10 5
4 4
29 560483
32 1048983
13 365
2 2
31 1247237
43 375234170
36 7231281
22 18830
41 149705195
26 9557
6 9
27 7652
11 110
30 878925
38 41219919
8 10
37 37400625
38 44519160
11 32
44 209735548
18 232
24 7476
16 62
18 1065
36 12788273
17 985
19 1496
23 19993
45 1318092570
6 5
30 1058723
19 4354
15 65
40 150111533
32 828452
25 36106
7 20
40 149164451
40 86238515
21 772
13 278
30 336076
18 2395
29 601610
43 510924307
17 968
23 42442
32 318071
14 585
43 178089030
1 1
24 36994
34 2734401
