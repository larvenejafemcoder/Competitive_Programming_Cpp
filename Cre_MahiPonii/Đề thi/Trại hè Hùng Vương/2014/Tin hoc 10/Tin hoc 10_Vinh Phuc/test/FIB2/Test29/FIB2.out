3889
99743
97548
187
39
87021
43337
3752670
85761095
29391093
1185
260
382
13119762
12
194730
5
738640
633713
38895568
3101396
38688347
4980067
48412
784
39693827
242598
6450747
7469601
5
13343664
0
365151815
684940576
58567
580
2
763
49
154309
1
7
1
2
2
214086
400676
140
1
476402
143326699
2762104
7193
57182296
3651
4
2923
42
335720
15744608
4
14285768
17004806
12
80111851
89
2856
24
407
4884686
376
572
7637
503466562
2
404396
1663
25
57337504
316441
13791
8
56975750
32940182
295
106
128370
915
229795
195155720
370
16212
121492
224
68023957
0
14131
1044448
